modules/RTCStats/RTCStats.ts

```diff
@@ -34,19 +34,19 @@ function connectionFilter(config) {
  * Config and conference changes are handled by the start method.
  */
 class RTCStats {
-    private _initialized: boolean = false;
-    private _trace: any = null;
+    _initialized = false;
```

Comment:
Nit: revert to using private, and it seems boolean type was removed.

Reply:
Right away. Reverted back to using the private modifier and type assertion on boolean.